src/main/java/com/twilio/jwt/taskrouter/Policy.java

```diff
@@ -17,6 +18,7 @@
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
+@JsonPropertyOrder({"allowed","method","postFilter","queryFilter","url"})
```

Comment:
update this to @JsonPropertyOrder(alphabetic = true) . This way we don't need to keep track of new properties being added.

Reply:
Thank you! Updated the PR!